xknx/devices/remote_value.py

```diff
@@ -28,16 +29,22 @@ def __init__(self,
 
         self.group_address = group_address
         self.group_address_state = group_address_state
-        self.after_update_cb = after_update_cb
+        self.sync_state = sync_state
         self.device_name = "Unknown" \
             if device_name is None else device_name
+        self.after_update_cb = after_update_cb
         self.payload = None
 
     @property
     def initialized(self):
         """Evaluate if remote value is initialized with group address."""
         return bool(self.group_address_state or self.group_address)
 
+    @property
+    def readable(self):
```

Comment:
Hmm, do we find a more explicit name for this property? Not having any better suggestion, but I think this value is not self explaining.

Reply:
I don't have a better name too. 
I think in the context of this module - the direct representation of a KNX Communication object - it's not that bad. I think of it as the pendant to the KNX Read Flag of a GA.